File: pset6/cash.py
from cs50 import get_float

change = 0.00

while change <= 0:
    change = get_float("Change owed: ") * 100

coin = 0

while change != 0:
    if change >= 25:
        coin += 1
        change -= 25
    elif change >= 10:
        coin += 1
        change -= 10
    elif change >= 5:
        coin += 1
        change -= 5
    else:
        coin += 1
        change -= 1
print(coin)

File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    
    int cents;
    int coins = 0;
    int quarter = 25;
    int dime = 10;
    int nickel = 5;
    int pennie = 1;
    float change;
    
    //trava
    do
    {
        change = get_float("Change owed: ");
    }
    while (change <= 0);
    
    //contador de moedas
    cents = round(change * 100);
    
    do
    {
        if (cents >= quarter)
        {
            cents = cents - quarter;
            coins++;
        }
        else if (cents >= dime && cents < quarter)
        {
            cents = cents - dime;
            coins++;
        }
        else if (cents >= nickel && cents < dime)
        {
            cents = cents - nickel;
            coins++;
        }
        else
        {
            cents = cents - pennie;
            coins++;
        }
        
    }
    while (cents != 0);
    printf("%i\n", coins);
}